README.md

```diff
@@ -17,3 +17,10 @@ or simply:
     -DEnv=preprod => This will allow the tests to be ran against the preprod enviroment. 
     
     If the number of cases need adjusting, this can be done by altering .body(ElFileBody("bodies/25CaseXMLMessage.xml")).asXml in XMLEndpointAPI file. Changing '25CaseXMLMessage.xml' to '100CaseXMLMessage.xml'. This will allow a bigger message for more intense load test purposes. 
+    
+    
+    The test has a mock live data databank availble for users that dont have live data on there machine to be able to test. 
+    To test simply go to the /config/Data direcotry and uncomment "val csvDefendantMatchMock = csv("data/defendantMatch_Mock.csv").random" out. 
```

Comment:
The CSV in use needs to default to something simple with fake data but which would allow the code to run out of the box. There could be a command line parameter to allow the user to point to a different CSV for situations where they are running with real data. The user shouldn't need to alter code to get this to work.